.github/workflows/publish-extension.yaml

```diff
@@ -46,12 +46,21 @@ jobs:
         run: |
           # Use the override value by preference
           VERSION=${INPUT_VERSION}
-          # If no valid override input was entered, then try to use the release tag
-          if [[ ! $VERSION =~ ^[0-9.]+ ]]; then
+          if [[ $VERSION =~ ^[0-9.]+$ ]]; then
+            echo "Using the valid override input and setting version to $VERSION"
+          else
+            # If no valid override input was entered, then try to use the release tag
             VERSION=${TAG_VERSION}
+            if [[ $VERSION =~ ^[0-9.]+$ ]]; then
```

Comment:
Actually as this is the override, we shouldn't be so strict, as we sometimes will want to set a version "3.4-TEST" in the override, I imagine? So I could remove the `$` at end of this regex, like below line 62.